src/main/kotlin/no/nav/familie/ef/søknad/interceptor/ApiKeyInjectingClientInterceptor.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.familie.ef.søknad.interceptor
+
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpRequest
+import org.springframework.http.client.ClientHttpRequestExecution
+import org.springframework.http.client.ClientHttpRequestInterceptor
+import org.springframework.http.client.ClientHttpResponse
+
+import java.io.IOException
+import java.net.URI
+
+class ApiKeyInjectingClientInterceptor(private val apiKeys: Map<URI, Pair<String, String>>) : ClientHttpRequestInterceptor {
```

Comment:
Nice :D Er det sånn at denne sjekken skal gjøres på alle http-requests? Bør vi i så fall også implementere dette som filter, for å være konsekvente?